### DISCUSSION TITLE
Spacial Editor - Focus Selection should focus on remote nodes

### DISCUSSION THREAD

**Discussion by allenwp** [godotengine/godot-proposals#6914] [Editor]
**Proposal:** Focus Selection action (pressing F) should focus on selected remote nodes, similar to how it focuses on selected local nodes.

**Use case:** I have a game that involves a player that moves quickly through the scene. When enabling "Project Camera Override" in the 3D spacial editor, I often want to see where the player is in the scene, but when I press F to focus selection on the remote player node, it instead focuses on the previously selected local node.

**Current behaviour:**

1. Select remote node
2. Inspector panel shows newly selected remote node
3. 3D spacial editor shows previously selected local node
4. Focus selection action (pressing F) focuses on previously selected local node

**Proposed behaviour:**

1. Select remote node
2. Inspector panel shows newly selected remote node
3. _Change:_ Previously selected local node is entirely deselected and no longer shows as selected in the 3D spacial editor
4. _Change:_ Focus selection action (pressing F) focuses on newly selected remote node

Point 3 in the proposed behaviour also addresses a usability issue with selecting remote nodes: With the current behaviour, there is a discrepancy between the 3D spacial editor view (which shows the previously selected local node) and the inspector panel (which shows the newly selected remote node).

With the proposed behaviour, this usability issue would no longer be a problem because the 3D spacial editor would not show a different node than the inspector panel.

**Edit: I've moved this to a proposal issue: #8130**

**Comment by allenwp** [comment]
I believe that https://github.com/godotengine/godot-proposals/issues/7213 could implement and solve the issues described in this discussion.

**Reply by Zireael07** [reply]
... which is an extremely huge undertaking that will likely take years to materialize. Yes it would make this issue irrelevant, but this is a separate, much smaller usability issue

**Comment by allenwp** [comment]
Closing as superseded by proposal #8130
